apps/bors_worker/lib/batcher.ex

```diff
@@ -516,4 +545,11 @@ defmodule BorsNG.Worker.Batcher do
   defp get_repo_conn(project) do
     Project.installation_connection(project.repo_xref, Repo)
   end
+
+  defp put_incomplete_on_hold(batch) do
```

Comment:
The biggest missing feature is that it doesn't do anything about the status as shown on the GitHub PR page. It's still going to say that it's "Running," when it should probably change the commit status so that it says it's been put on hold.